runtime/vm/jvminit.c

```diff
@@ -5764,6 +5764,27 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 		Trc_VM_contendedLinesizeFailed(queryResult);
 	}
 
+	/* check processor support for cache writeback */
+#if defined(J9X86) || defined(J9HAMMER)
+	J9ProcessorDesc desc;
+	j9sysinfo_get_processor_description(&desc);
+	/* cache line size in bytes is the value of bits 8-15 * 8 */
+	U_32 lineSize = ((desc.features[2] & 0xFF00) >> 8) * 8;
+	vm->cpuCacheLineSize = lineSize;
+	if (j9sysinfo_processor_has_feature(&desc, J9PORT_X86_FEATURE_CLWB)) {
+		vm->cpuCacheWritebackCapabilities = J9PORT_X86_FEATURE_CLWB;
+	} else if (j9sysinfo_processor_has_feature(&desc, J9PORT_X86_FEATURE_CLFLUSHOPT)) {
+		vm->cpuCacheWritebackCapabilities = J9PORT_X86_FEATURE_CLFLUSHOPT;
+	} else if (j9sysinfo_processor_has_feature(&desc, J9PORT_X86_FEATURE_CLFSH)) {
+		vm->cpuCacheWritebackCapabilities = J9PORT_X86_FEATURE_CLFSH;
+	} else {
+		vm->cpuCacheWritebackCapabilities = 0;
+	}
+#else
```

Comment:
Might look better to zero initialize these always, then update them under the X86 ifdef.